myapp/scheduler/scheduler.py

```diff
@@ -1,9 +1,9 @@
 from apscheduler.schedulers.background import BackgroundScheduler
 from .stock_api_update import stock_api_update
-from django.core import management
 
 
 def start():
     scheduler = BackgroundScheduler()
+    scheduler.add_job(stock_api_update)
     scheduler.add_job(stock_api_update, 'interval', seconds=10)
```

Comment:
Why has the job been added twice?

Reply:
hi Kamal , i've add comments  on slack , i've explained why i did this